src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/kodilla_project?serverTimezone=Europe/Warsaw&useSSL=False
+spring.datasource.url=jdbc:mysql://localhost:3306/kodilla_project?serverTimezone=Europe/Warsaw&useSSL=False&allowPublicKeyRetrieval=true

```

Comment:
bez tej adnotacji miałem błąd: [Public Key Retrieval is not allowed](https://stackoverflow.com/questions/50379839/connection-java-mysql-public-key-retrieval-is-not-allowed)